python/utils/config.py

```diff
@@ -13,6 +13,8 @@ class Config(BaseSettings):
     grpc_data_stream_endpoint: str
     grpc_data_stream_api_key: str
     db_connection_uri: str
+    health_port: int = 8080
```

Comment:
Can you also update `config.yaml.example`?

Reply:
done